content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: একটি ক্লাস্টার তৈরি করতে Minikube ব্যবহার করে
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="bn">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+      <div class="col-md-8">
+          <h3>Objectives</h3>
+                <ul>
+                    <li>Kubernetes ক্লাস্টার কি তা জানুন.</li>
+                    <li>Minikube কি তা জানুন.</li>
+                    <li>একটি অনলাইন টার্মিনাল ব্যবহার করে একটি Kubernetes ক্লাস্টার শুরু করুন।</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Kubernetes Clusters</h3>
+                <p>
+                <b>কুবারনেটস কম্পিউটারের একটি অত্যন্ত উপলব্ধ ক্লাস্টার সমন্বয় করে যা একটি একক হিসাবে কাজ করার জন্য সংযুক্ত।</b> Kubernetes-এর বিমূর্ততা আপনাকে বিশেষভাবে পৃথক মেশিনে না বেঁধে একটি ক্লাস্টারে কন্টেইনারাইজড অ্যাপ্লিকেশন স্থাপন করতে দেয়। স্থাপনার এই নতুন মডেলটি ব্যবহার করার জন্য, অ্যাপ্লিকেশনগুলিকে এমনভাবে প্যাকেজ করা দরকার যাতে সেগুলিকে পৃথক হোস্ট থেকে আলাদা করা যায়: তাদের কন্টেইনারাইজ করা দরকার। কন্টেইনারাইজড অ্যাপ্লিকেশনগুলি অতীতের স্থাপনার মডেলগুলির তুলনায় আরও নমনীয় এবং উপলব্ধ, যেখানে অ্যাপ্লিকেশনগুলি সরাসরি হোস্টের সাথে গভীরভাবে একত্রিত প্যাকেজ হিসাবে নির্দিষ্ট মেশিনে ইনস্টল করা হয়েছিল। <b>Kubernetes একটি ক্লাস্টার জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির বিতরণ এবং সময়সূচীকে আরও দক্ষ উপায়ে স্বয়ংক্রিয় করে।</b> Kubernetes একটি ওপেন সোর্স প্ল্যাটফর্ম এবং এটি উৎপাদনের জন্য প্রস্তুত।
+                </p>
+                <p>>A কুবারনেটস ক্লাস্টার দুটি ধরণের সংস্থান নিয়ে গঠিত:
+
+                    <ul>
+                        <li>The <b>Control Plane</b> coordinates the cluster</li>
+                        <li><b>Nodes</b> are the workers that run applications</li>
```

Comment:
Kindly translate these words too. `Nodes`, `are the workers that run applications`